docs/git/cheat-sheet.md

```diff
@@ -1,9 +1,55 @@
-# Git Cheat Sheet (Detailed)
+# Git Cheat Sheet
 
-See also
-[fixing commits][fix-commit]
+See also [fixing commits][fix-commit]
 
-Commands:
+## Common Commands
+
+- add
+    - `git add foo.py`
+- checkout
+    - `git checkout -b new-branch-name`
+    - `git checkout master`
+    - `git checkout old-branch-name`
+- commit
+    - `git commit --amend`
```

Comment:
Replace with an example that does a commit eg `git commit -m "section: Commit title.".